external-builds/pytorch/pytorch_vision_repo.py

```diff
@@ -48,6 +48,20 @@
 THIS_DIR = Path(__file__).resolve().parent
 THIS_PATCHES_DIR = THIS_DIR / "patches" / THIS_MAIN_REPO_NAME
 
+(
+    DEFAULT_ORIGIN,
+    DEFAULT_HASHTAG,
+    DEFAULT_PATCHSET,
+    HAS_RELATED_COMMIT,
+) = repo_management.read_pytorch_rocm_pins(
+    THIS_DIR / "pytorch",
+    os="centos",
```

Comment:
Maybe we should update the related_commits format to get rid of the OS component? I can't recall the last time we maintained different commits for Ubuntu vs CentOS. Also, with CentOS builds deprecated, this probably doesn't age well.